app/containers/ViewQuestion/ContentRating.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { getFormattedNum } from 'utils/numbers';
+import MarkAsAcceptedIcon from './MarkAsAcceptedIcon';
+
+/* eslint consistent-return: "error" */
+const isRatingChanged = (history, account, flag) => {
```

Comment:
I think calculation should happen in questionManagement.js, getQuestionData function. Similar to calculation of field `y.isItWrittenByMe = user === y.user;` can calculate `upvotedByMe` and `downvotedByMe`.